app/views/admin/teams/index.html.haml

```diff
@@ -0,0 +1,17 @@
+.container.my-4
+  .row
+    .col-12
+      .table-responsive
+        %table.table.table-bordered
+          %thead
+            %tr
+              %th= t('activerecord.models.user_group.name')
```

Comment:
Just for consistency with the rest of the code (e.g. app/views/admin/user_groups/new.html.haml), might want to use 
UserGroup.human_attribute_name('name') 
instead here

Reply:
actually I'm not convinced to that type of "localisation" because it works also for UserGroup.human_attribute_name('foo') so for the attributes which does not exist for the model too.
Also, in my version we are sure that field name will have the same label in form and in the table, so consistent.